src/ert/parsing/config_keywords.py

```diff
@@ -5,8 +5,17 @@
 
 from pydantic import BaseModel
 
-from .config_errors import ConfigValidationError
-from .lark_parser_types import FileContextToken, Instruction
+from . import ConfigValidationError
```

Comment:
should use local import from config_errors.py ?

Reply:
oops, fixed